allocations/views.py

```diff
@@ -55,9 +56,22 @@ def get_all_alloc(request):
         json: dumps all data as serialized json.
     """
     try:
+        keycloak_client = KeycloakClient()
+        user_attributes = keycloak_client.get_all_users_attributes()
```

Comment:
We only need the PI's user attributes here, why does this fetch for all users? Seems like that could be expensive.

Reply:
this is for the allocation admin page, which will load all projects/allocations, and we'll need the attributes for all the PIs.

Reply:
Ha! I see your point - getting users with the pi role. will fix